js/listanchors.js

```diff
@@ -88,6 +88,65 @@ function listanchors()
         id += key;
         var e = document.getElementById(id);
         e.innerHTML = newText;
+
+    }
+    function addVersionSelector() {
+      // Latest version offered by the archive builds
+      // This needs to be manually updated after new versions have been archived
+      var currentArchivedVersion = 74;
+      // build URLs for dlang.org: DDoc + Dox
+      var ddocModuleURL = document.body.id.replace(/[.]/g, "_") + ".html";
+      var ddoxModuleURL = document.body.id.replace(/[.]/g, "/") + ".html";
+
+      // build list of versions available in the docarchives
+      var archivedVersions = [];
+      while (currentArchivedVersion >= 66) {
+        archivedVersions.push("2.0" + currentArchivedVersion--);
+      }
+      archivedVersions = archivedVersions.map(function(e) {
+          return {
+            name: e,
+            url: "https://docarchives.dlang.io/v" + e + ".0/phobos/" + ddocModuleURL,
+            selected: false,
+          };
+      });
+
+      var rootURL = location.href.split(/\/(phobos|library)(-prerelease)?/)[0]
+      var onlineVersions = [{
+        name: "master",
+        url: rootURL + "/phobos-prerelease/" + ddocModuleURL,
+      },{
+        name: "master (beta)",
+        url: rootURL + "/library-prerelease/" + ddoxModuleURL,
+      },{
+        name: "stable",
+        url: rootURL + "/phobos/" + ddocModuleURL,
+      },{
+        name: "stable (beta)",
+        url: rootURL + "/library/" + ddoxModuleURL,
+      }];
+
+      // set the current URL as selected
+      var currentURL = location.href.split(/[#?]/)[0];
+      onlineVersions.forEach(function(v, i) {
+        onlineVersions[i].selected = v.url === currentURL;
+      });
+
+      // build select box of all versions and append to current DOM
+      var versions = onlineVersions.concat(archivedVersions);
+      var options = versions.map(function(e, i){
+        return "<option value='" + i + "'" + (e.selected ? "selected" : "") + ">" + e.name + "</option>";
+      });
+      $("h1").after("<div class='version-changer-container fa-select'><select id='version-changer'>" + options.join("") + "</select></div>");
```

Comment:
I'm pretty sure you're supposed to put the element you're floating *before* the element it is supposed to "overlap", not `after`.

Why does this work for you?